kythe/go/platform/tools/kzip/scancmd/scancmd.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package createcmd provides the kzip command for creating simple kzip archives.
+package scancmd
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"time"
+
+	"kythe.io/kythe/go/platform/kzip"
+	"kythe.io/kythe/go/util/cmdutil"
+
+	"github.com/google/subcommands"
+)
+
+type scanCommand struct {
+	cmdutil.Info
+
+	input string
+}
+
+// New creates a new subcommand for merging kzip files.
+func New() subcommands.Command {
+	return &scanCommand{
+		Info: cmdutil.NewInfo("scan", "scan single kzip archive", "--input path"),
```

Comment:
Probably just call this `count`.  Is this really needed?  Seems like it is just for testing and shouldn't be in the `kzip` utility.